src/lib/client.ts

```diff
@@ -472,13 +477,44 @@ export class CustomClient extends Client {
     if (targetChannelId) filter.targetChannelId = targetChannelId;
     await AutoPingModel.deleteMany(filter);
   }
-
+  u;
   async getAllAutoPing(guildId: string): Promise<IAutoPing[]> {
     return await AutoPingModel.find({
       guildId: guildId,
     });
   }
 
+  async addEmoji(guildId: string, name: string): Promise<void> {
+    const millis15min = 15 * 60 * 1000;
+    const time = Date.now();
+    const emojiUsage =
+      (await EmojiUsageModel.findOne({ guildId: guildId, name: name })) ??
+      (await EmojiUsageModel.create({
+        guildId: guildId,
+        name: name,
+        count: 0,
+        lastUsage: null,
+      }));
```

Comment:
Read and implement [this stackoverflow answer](https://stackoverflow.com/a/49830816) instead.